interpretation/ww_head.py

```diff
@@ -133,8 +134,16 @@ def process(message, isGameMaster = False):
         # This command allows users to view information about a conspiracy channel.
         # Says the user must be in a cc if they're not.
         if is_command(message,['info']):
-            # TODO
-            return todo()
+            guild = message.channel.guild
+            owner_id = channel_get(message.channel.id,'owner')
+            owner_object = guild.get_member(int(owner_id))
+            embed = Embed(color=0x00cdcd, title='Conspiracy Channel Info')
+            embed.add_field(name='Channel Owner', value='<@' + owner_id + '>')
+            embed.add_field(name='Channel Name', value=message.channel.name)
+            embed.add_field(name='Participants', value='[Bob Roberts], [Dummy], [Randium], [BenTechy66], [Ed588]')
+            embed.set_footer(text='Conspiracy Channel Information requested by ' + message.author.nick)
+            embed.set_thumbnail(url=owner_object.avatar_url)
```

Comment:
You do not have a backup plan if owner_object is None, which happens in the case that the owner has left the server, or if the message was a DM to the bot.  In the first case, you could back it up by showing that cult member picture I gave to you, and the other, where owner_id already returns None, you should add an error mailbox saying they cannot use the command in their current channel.